File: src/game/Game.java
package game;

import game.world.World;

import static org.lwjgl.opengl.GL11.GL_COLOR_BUFFER_BIT;
import static org.lwjgl.opengl.GL11.glClear;

public class Game extends GameLoop {

    World world;

    public void init() {
        super.init();

        world = new World();
    }

    public void update(long delta) {
        world.update(delta);
    }

    public void render() {
        glClear(GL_COLOR_BUFFER_BIT);

        world.render();
    }

    public static void main(String[] args) {
        GameLoop game = new Game();
    }
}

File: src/game/Game.kt
package game

import game.world.World

import org.lwjgl.opengl.GL11.GL_COLOR_BUFFER_BIT
import org.lwjgl.opengl.GL11.glClear

class Game : GameLoop() {

    lateinit internal var world: World

    override fun init() {
        super.init()

        world = World()
    }

    override fun update(delta: Long) {
        world.update(delta)
    }

    override fun render() {
        glClear(GL_COLOR_BUFFER_BIT)

        world.render()
    }

    companion object {

        @JvmStatic fun main(args: Array<String>) {
            val game = Game()
        }
    }
}
